duffel_api/api/duffel_payments/payment_intents.py

```diff
@@ -53,14 +53,14 @@ def __init__(self, client):
         self._amount = None
         self._currency = None
 
-    def _validate_payment_keys(payment_details):
+    def _validate_payment_keys(self, payment_details):
```

Comment:
This is a static method which I forgot to tag as such. I don't think that there's a need to make it an instance method.

Reply:
Yeah, this just needs the `@staticmethod`, not `self`.

Reply:
I've added the missing decorator in 896f664. Apologies for missing it previously!